### ISSUE TITLE
this error pitest with ant ,use the hcoles/pitest-ant-example.

### ISSUE THREAD

**Issue by redjiang** [hcoles/pitest#316]
pit:
   [pitest] 14:03:07 PIT >> INFO : Verbose logging is disabled. If you encounter an problem please enable it before reporting an issue.
   [pitest] 14:03:07 PIT >> INFO : MINION : ����: �Ҳ������޷��������� org.pitest.coverage.execute.CoverageMinion

**Comment by hcoles** [comment]
These are not error messages - these are info log messages.

Does pitest successfully produce results in the `pitReports` directory?

**Comment by redjiang** [comment]
thank you for your answer!
yes,only 	empty directory!

**Comment by hcoles** [comment]
Could you post the rest of the log output?

**Comment by redjiang** [comment]
build directory has three directories(classes、test-classes、test-result),and these directories all have content.
The message:
Buildfile: C:\mooctest\projects\110\12253\Schedule\build.xml
clean:
   [delete] Deleting directory C:\mooctest\projects\110\12253\Schedule\build
compile:
    [mkdir] Created dir: C:\mooctest\projects\110\12253\Schedule\build\classes
    [javac] Compiling 4 source files to C:\mooctest\projects\110\12253\Schedule\build\classes
test:
    [mkdir] Created dir: C:\mooctest\projects\110\12253\Schedule\build\test-result
    [mkdir] Created dir: C:\mooctest\projects\110\12253\Schedule\build\test-classes
    [javac] Compiling 2 source files to C:\mooctest\projects\110\12253\Schedule\build\test-classes
[junitreport] Processing C:\mooctest\projects\110\12253\Schedule\build\test-result\TESTS-TestSuites.xml to C:\Users\ADMINI~1\AppData\Local\Temp\null188120935
[junitreport] Loading stylesheet jar:file:/D:/Program%20Files/eclipse/plugins/org.apache.ant_1.9.6.v201510161327/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 696ms
[junitreport] Deleting: C:\Users\ADMINI~1\AppData\Local\Temp\null188120935
pit:
   [pitest] 14:03:07 PIT >> INFO : Verbose logging is disabled. If you encounter an problem please enable it before reporting an issue.
   [pitest] 14:03:07 PIT >> INFO : MINION : ����: �Ҳ������޷��������� org.pitest.coverage.execute.CoverageMinion

**Comment by redjiang** [comment]
Buildfile: G:\eclipse_project\pit-ant-example\build.xml
clean:
   [delete] Deleting directory G:\eclipse_project\pit-ant-example\build
compile:
    [mkdir] Created dir: G:\eclipse_project\pit-ant-example\build\classes
    [javac] Compiling 2 source files to G:\eclipse_project\pit-ant-example\build\classes
test:
    [mkdir] Created dir: G:\eclipse_project\pit-ant-example\build\test-result
    [mkdir] Created dir: G:\eclipse_project\pit-ant-example\build\test-classes
    [javac] Compiling 1 source file to G:\eclipse_project\pit-ant-example\build\test-classes
[junitreport] Processing G:\eclipse_project\pit-ant-example\build\test-result\TESTS-TestSuites.xml to C:\Users\ADMINI~1\AppData\Local\Temp\null1765915442
[junitreport] Loading stylesheet jar:file:/D:/Program%20Files/eclipse/plugins/org.apache.ant_1.9.6.v201510161327/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 996ms
[junitreport] Deleting: C:\Users\ADMINI~1\AppData\Local\Temp\null1765915442
pit:
   [pitest] 19:53:15 PIT >> INFO : Verbose logging is disabled. If you encounter an problem please enable it before reporting an issue.
   [pitest] 19:53:16 PIT >> INFO : MINION : ����: �Ҳ������޷��������� org.pitest.coverage.execute.CoverageMinion

**Comment by redjiang** [comment]
Buildfile: G:\eclipse_project\pit-ant-example\build.xml
clean:
   [delete] Deleting directory G:\eclipse_project\pit-ant-example\build
compile:
    [mkdir] Created dir: G:\eclipse_project\pit-ant-example\build\classes
    [javac] Compiling 3 source files to G:\eclipse_project\pit-ant-example\build\classes
test:
    [mkdir] Created dir: G:\eclipse_project\pit-ant-example\build\test-result
    [mkdir] Created dir: G:\eclipse_project\pit-ant-example\build\test-classes
    [javac] Compiling 2 source files to G:\eclipse_project\pit-ant-example\build\test-classes
[junitreport] Processing G:\eclipse_project\pit-ant-example\build\test-result\TESTS-TestSuites.xml to C:\Users\ADMINI~1\AppData\Local\Temp\null535087826
[junitreport] Loading stylesheet jar:file:/D:/Program%20Files/eclipse/plugins/org.apache.ant_1.9.6.v201510161327/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 838ms
[junitreport] Deleting: C:\Users\ADMINI~1\AppData\Local\Temp\null535087826
pit:
   [pitest] 19:58:09 PIT >> INFO : Verbose logging is disabled. If you encounter an problem please enable it before reporting an issue.
   [pitest] 19:58:09 PIT >> INFO : MINION : ����: �Ҳ������޷��������� org.pitest.coverage.execute.CoverageMinion

**Comment by hcoles** [comment]
I take it there is no more output after this?

I've occasionally seen issues in the past with virus checkers on windows preventing communication between the main and child processes (its over a socket).

If you have one try running with it disabled and see if you get some output.

**Comment by redjiang** [comment]
yes ,the eclipse console is no more output ! i  try running with it disabled but the output is still only those! 
the last output some times is slave： PIT >> INFO : SLAVE : ����: �Ҳ������޷��������� org.pitest.coverage.execute.CoverageSlave
All of the console information：
Buildfile: D:\Users\lenovo\workspace\pitest-ant-example\build.xml
clean:
   [delete] Deleting directory D:\Users\lenovo\workspace\pitest-ant-example\build
compile:
    [mkdir] Created dir: D:\Users\lenovo\workspace\pitest-ant-example\build\classes
    [javac] Compiling 3 source files to D:\Users\lenovo\workspace\pitest-ant-example\build\classes
test:
    [mkdir] Created dir: D:\Users\lenovo\workspace\pitest-ant-example\build\test-result
    [mkdir] Created dir: D:\Users\lenovo\workspace\pitest-ant-example\build\test-classes
    [javac] Compiling 2 source files to D:\Users\lenovo\workspace\pitest-ant-example\build\test-classes
[junitreport] Processing D:\Users\lenovo\workspace\pitest-ant-example\build\test-result\TESTS-TestSuites.xml to C:\Users\lenovo\AppData\Local\Temp\null1737279757
[junitreport] Loading stylesheet jar:file:/C:/Users/lenovo/.p2/pool/plugins/org.apache.ant_1.9.6.v201510161327/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 872ms
[junitreport] Deleting: C:\Users\lenovo\AppData\Local\Temp\null1737279757
clean:
   [delete] Deleting directory D:\Users\lenovo\workspace\pitest-ant-example\build
compile:
    [mkdir] Created dir: D:\Users\lenovo\workspace\pitest-ant-example\build\classes
    [javac] Compiling 3 source files to D:\Users\lenovo\workspace\pitest-ant-example\build\classes
clean:
   [delete] Deleting directory D:\Users\lenovo\workspace\pitest-ant-example\build
clean:
compile:
    [mkdir] Created dir: D:\Users\lenovo\workspace\pitest-ant-example\build\classes
    [javac] Compiling 3 source files to D:\Users\lenovo\workspace\pitest-ant-example\build\classes
test:
    [mkdir] Created dir: D:\Users\lenovo\workspace\pitest-ant-example\build\test-result
    [mkdir] Created dir: D:\Users\lenovo\workspace\pitest-ant-example\build\test-classes
    [javac] Compiling 2 source files to D:\Users\lenovo\workspace\pitest-ant-example\build\test-classes
[junitreport] Processing D:\Users\lenovo\workspace\pitest-ant-example\build\test-result\TESTS-TestSuites.xml to C:\Users\lenovo\AppData\Local\Temp\null1802379988
[junitreport] Loading stylesheet jar:file:/C:/Users/lenovo/.p2/pool/plugins/org.apache.ant_1.9.6.v201510161327/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 159ms
[junitreport] Deleting: C:\Users\lenovo\AppData\Local\Temp\null1802379988
pit:
   [pitest] 9:29:52 PIT >> INFO : Verbose logging is disabled. If you encounter an problem please enable it before reporting an issue.
   [pitest] 9:29:52 PIT >> INFO : SLAVE : ����: �Ҳ������޷��������� org.pitest.coverage.execute.CoverageSlave
